app/assets/javascripts/patients.coffee

```diff
@@ -37,6 +54,26 @@ ready = ->
   $(document).on "change", "#abortion-information-form input, #external_pledges input", ->
     updateBalance()
 
+  $(document).on "change", field_selectors, ->
+    i = 0
```

Comment:
Could we wrap this in its own named function? Something like `mark_fulfilled_when_fields_checked` or something similar that captures the spirit of what this does?